api-gateway/src/main/resources/static/scripts/auth/update-role-form/role-update.template.html

```diff
@@ -0,0 +1,12 @@
+<h2>Update User Role</h2>
+<form style="background-color: #f5f5f5; padding: 20px; border-radius: 10px; box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);">
+  <div class="form-group">
+    <label>Select Role:</label>
+    <div ng-repeat="role in roles" class="form-check">
+      <input type="checkbox" class="form-check-input" id="{{role}}" ng-model="selectedRoles[role]" ng-true-value="'{{role}}'" ng-false-value="undefined">
+      <label class="form-check-label" for="{{role}}">{{role}}</label>
+    </div>
+  </div>
```

Comment:
Why are we using checkboxes ? Radio buttons would fit better since we would want to choose only one role.